nemo/collections/asr/data/audio_to_text.py

```diff
@@ -204,6 +221,10 @@ def __init__(
         )
         self.featurizer = WaveformFeaturizer(sample_rate=sample_rate, int_values=int_values, augmentor=augmentor)
         self.trim = trim
+        self.return_sample_id = return_sample_id
+
+    def get_sample(self, sample_id):
```

Comment:
Rename to get_manifest_sample - first read it seems like get audio sample which  is wrong here.

Reply:
done.